### DISCUSSION TITLE
Parent project calling a child task ?

### DISCUSSION THREAD

**Discussion by marciocadev** [projen/projen#2258] [Q&A]
There's anyway to make the parent project run a child task ?

**Comment by pflorek** [comment]
Yes, either prepend, append or create a new task.

```typescript
project.tasks.tryFind("pre-compile")?.prependExec("(cd functions && yarn test)");
```

https://github.com/pepperize/cdk-apigateway-swagger-ui/blob/3a9957879a82ee85744910b025112ecd47781157/.projenrc.ts#L84

**Reply by marciocadev** [reply]
thank you

**Comment by mvs5465** [comment]
Just to add to @pflorek's comment, the builtin tasks can be referenced directly: 

```
project.tasks.preCompileTask.prependExec("(cd functions && yarn test)");
```
